src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -178,8 +178,6 @@ public void onTraversalStart() {
             case GENOMES:
                 vetTsvCreator = new VetTsvCreator(sampleIdentifierForOutputFileName, sampleId, tableNumberPrefix, outputDir);
                 break;
-            case ARRAYS:
```

Comment:
do we even need this switch statement? we make the same call for both exomes and genomes

Reply:
Yes!  I was restraining myself to clean up code and save that for the next PR.  Mostly to make the review of this PR easy (ie it's all structural, you don't need to look at each line of code) and the next one smaller, but where the lines of code matter.  We also have some "common" code that is now really tool specific that we can shift.  Things like the above, etc.